### PR TITLE
Update to version 3.1.1.2

### PR DESCRIPTION
**Pull request by alifahadk**

### CODE REVIEW

File: docs/lakehouse-sink.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ To build the Lakehouse sink connector from the source code, follow these steps.
 
    ```bash
    ls target
-   pulsar-io-lakehouse-{{connector:version}}.nar
+   pulsar-io-lakehouse-3.1.1.2.nar
```
Comment (ydmk) [inline docs/lakehouse-sink.md:45]: Does this need changing?

File: docs/lakehouse-source.md [hunk 2]
```diff
@@ -206,7 +206,7 @@ This example explains how to create a Lakehouse source connector in an on-premis
 1. Copy the NAR package of the Lakehouse source connector to the Pulsar connectors directory.
 
     ```
-    cp pulsar-io-lakehouse-{{connector:version}}.nar $PULSAR_HOME/connectors/pulsar-io-lakehouse-{{connector:version}}.nar
+    cp pulsar-io-lakehouse-3.1.1.2.nar $PULSAR_HOME/connectors/pulsar-io-lakehouse-3.1.1.2.nar
```
Comment (ydmk) [inline docs/lakehouse-source.md:209]: Probably don't need these changes, or they will at least need changing again if we update

File: pom.xml [hunk 3]
```diff
@@ -19,23 +19,18 @@
...
   <groupId>org.apache.pulsar.ecosystem</groupId>
   <artifactId>pulsar-io-lakehouse</artifactId>
-  <version>2.11.0-SNAPSHOT</version>
+  <version>3.1.1.2</version>
   <name>Pulsar Ecosystem :: IO Connector :: Lake House</name>
   <description>It is a lakehouse connector streaming convert data between lakehouse and Apache Pulsar.</description>
-
   <properties>
     <maven.compiler.source>8</maven.compiler.source>
     <maven.compiler.target>8</maven.compiler.target>
```
Comment (ydmk) [inline pom.xml:36]: The target probably needs to be changed to 17 at least

File: pom.xml [hunk 4]
```diff
@@ -45,11 +40,10 @@
     <redirectTestOutputToFile>true</redirectTestOutputToFile>
     <spotbugs-annotations.version>4.2.2</spotbugs-annotations.version>
     <testRetryCount>0</testRetryCount>
-
     <!-- connector dependencies -->
     <jackson.version>2.13.2.1</jackson.version>
     <lombok.version>1.18.22</lombok.version>
-    <pulsar.version>2.10.0.4</pulsar.version>
+    <pulsar.version>3.1.1.2</pulsar.version>
     <log4j2.version>2.17.2</log4j2.version>
```
Comment (ydmk) [inline pom.xml:47]: Update this to a version that's latest
I don't want to be hit by a log4j vulnerability

File: pom.xml [hunk 5]
```diff
@@ -45,11 +40,10 @@
...
     <slf4j.version>1.7.25</slf4j.version>
     <hadoop.version>3.2.4</hadoop.version>
```
Comment (ydmk) [inline pom.xml:49]: Remember to change this later

File: pom.xml [hunk 6]
```diff
@@ -62,12 +56,11 @@
     <aws.sdk.version>1.12.220</aws.sdk.version>
```
Comment (ydmk) [inline pom.xml:56]: Update this to v2 later

File: docs/lakehouse-source.md (hunk 2 above)
Comment (alifahadk) [reply docs/lakehouse-source.md:209]: When we merge a branch from the forked repo for a version update, these changes will be there again I guess

File: pom.xml (hunk 3 above)
Comment (alifahadk) [reply pom.xml:36]: Done

File: pom.xml (hunk 4 above)
Comment (alifahadk) [reply pom.xml:47]: Done

File: pom.xml (hunk 5 above)
Comment (alifahadk) [reply pom.xml:49]: updated to 3.3.1

**Comment by alifahadk** [comment]
Superseded by https://github.com/din-global/data-pulsar-io-lakehouse/pull/5

**Pull request closed without merging**